docs/src/main/tut/user-guide.md

```diff
@@ -441,7 +441,7 @@ For all `String`-based endpoints, Finch provides an `as[A]` method to perform ty
 is available for any `Endpoint[String]`, `Endpoint[Option[String]]` or `Endpoint[Seq[String]]` as
 long as a matching implicit `io.finch.DecodeEntity[A]` type-class is in the scope.
 
-This facility is designed to be intuitive, meaning that you do not have to provide a
+This facility is designed to be intuitive, meaning that you do not have to provide an
```

Comment:
So should I remove `io.finch` in the entire doc? in this PR?
Though there seem to be some exceptions like [`io.finch.circe.dropNullKeys._` here](https://finagle.github.io/finch/best-practices.html#picking-a-json-library), and maybe some other places that could be ambiguous?